src/transformers/cache_utils.py

```diff
@@ -1179,25 +1179,17 @@ class StaticCache(Cache):
 
     is_compileable = True
 
-    # TODO (joao): remove `=None` in non-optional arguments in v4.46. Remove from `OBJECTS_TO_IGNORE` as well.
```

Comment:
in `/utils/check_docstring.py`, there is a `OBJECTS_TO_IGNORE`. Let's remove the four caches from that list :)

(At the moment, we are skipping a few automated checks on these classes. By removing them from `OBJECTS_TO_IGNORE`, we re-enable the checks)

Reply:
Removed